src/core_modules/capture-core/components/ListView/Menu/ListViewMenu.component.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import React, { useCallback, memo, type ComponentType } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { IconButton, Paper, MenuList, MenuItem, Divider } from '@material-ui/core';
+import { MoreHoriz } from '@material-ui/icons';
+import Popper from '../../Popper/Popper.component';
+import type { Props } from './listViewMenu.types';
+
+const getStyles = () => ({
+    subHeader: {
+        paddingLeft: 16,
+        paddingRight: 16,
+        paddingTop: 10,
+        paddingBottom: 10,
+        color: '#717c8b',
+        fontWeight: 500,
+        fontSize: 16,
+        '&:focus': {
+            outline: 'none',
+            color: 'black',
+        },
+    },
+    subHeaderDivider: {
+        '&:focus': {
+            outline: 'none',
+            backgroundColor: 'black',
+        },
+    },
+});
+
+const ListViewMenuPlain = ({ customMenuContents = [], classes }: Props) => {
+    const renderPopperAction = useCallback((togglePopper: Function) => (
+        <IconButton onClick={togglePopper}>
+            <MoreHoriz />
+        </IconButton>
+    ), []);
+
+    const renderMenuItems = useCallback((togglePopper: Function) =>
+        customMenuContents
+            .map((content) => {
+                if (content.subHeader) {
+                    return [
+                        <Divider
+                            key={`${content.key}divider`}
+                            data-test={`subheader-divider-${content.key}`}
+                            className={classes.subHeaderDivider}
+                        />,
+                        <div
+                            key={content.key}
+                            data-test={`subheader-${content.key}`}
+                            className={classes.subHeader}
+                        >
+                            {content.subHeader}
+                        </div>,
+                    ];
+                }
+
+                return (
+                    <MenuItem
+                        key={content.key}
+                        data-test={`menu-item-${content.key}`}
+                        onClick={() => {
+                            if (!content.clickHandler) {
+                                return;
+                            }
+                            togglePopper();
+                            // $FlowFixMe Using exact types, in my book this should work. Please tell me what I'm missing.
+                            content.clickHandler();
+                        }}
+                        // $FlowFixMe Using exact types, in my book this should work. Please tell me what I'm missing.
+                        disabled={!content.clickHandler}
+                    >
+                        {
+                            // $FlowFixMe Using exact types, in my book this should work. Please tell me what I'm missing.
+                            content.element}
+                    </MenuItem>
+                );
+            })
```

Comment:
Flow wants union types to be in if else statement _but_ with our eslint configuration if you write 
```
if (statement){
 return 1
} else {
 return 2
}

```
and you press save it will become 
```
if (statement){
 return 1
} 
return 2
```

Now flow wont be able to tell the union types apart anymore. I know its sad.

Perhaps you could do like so 
```
        customMenuContents
            .map((content) => {
                if (content.subHeader) {
                    return [
                        <Divider
                            key={`${content.key}divider`}
                            data-test={`subheader-divider-${content.key}`}
                            className={classes.subHeaderDivider}
                        />,
                        <div
                            key={content.key}
                            data-test={`subheader-${content.key}`}
                            className={classes.subHeader}
                        >
                            {content.subHeader}
                        </div>,
                    ];
                } else if (content.element) {
                    return (
                        <MenuItem
                            key={content.key}
                            data-test={`menu-item-${content.key}`}
                            onClick={() => {
                                togglePopper();
                                content.clickHandler && content.clickHandler();
                            }}
                            disabled={!content.clickHandler}
                        >
                            {content.element}
                        </MenuItem>
                    );
                }
                return null;
            })

```

Reply:
Thanks for digging into it.

Reply:
What is your plan in this case?

Reply:
Leaving this one as it is. We can't spend time refactoring code because flow is not keeping up. We need flow to help us find problems. If flow is just annoying and don't help us we should ignore it. If we can't ignore it, we need to remove it altogether.